src/main/kotlin/com/jetbrains/snakecharm/inspections/SmkMultilineFunctionCallInspection.kt

```diff
@@ -0,0 +1,76 @@
+package com.jetbrains.snakecharm.inspections
+
+import com.intellij.codeInspection.*
+import com.intellij.lang.ASTNode
+import com.intellij.openapi.command.WriteCommandAction
+import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiElement
+import com.intellij.psi.TokenType
+import com.intellij.psi.util.elementType
+import com.jetbrains.python.PyElementTypes
+import com.jetbrains.python.PyTokenTypes
+import com.jetbrains.python.psi.PyArgumentList
+import com.jetbrains.snakecharm.SnakemakeBundle
+import com.jetbrains.snakecharm.lang.psi.SmkRuleOrCheckpointArgsSection
+
+class SmkMultilineFunctionCallInspection : SnakemakeInspection() {
+    override fun buildVisitor(
+        holder: ProblemsHolder,
+        isOnTheFly: Boolean,
+        session: LocalInspectionToolSession
+    ) = object : SnakemakeInspectionVisitor(holder, session) {
+        override fun visitSmkRuleOrCheckpointArgsSection(st: SmkRuleOrCheckpointArgsSection) {
+            val args = st.children.firstOrNull { it is PyArgumentList } ?: return
+
+            if (multilineSectionDefinition(args as PyArgumentList)) {
+                return
+            }
+
+            val problemCalls = arrayListOf<PsiElement>()
+            args.children.filter { it.elementType == PyElementTypes.CALL_EXPRESSION }.forEach {
+                multilineFunctionCall(it.node, problemCalls)
+            }
+
+            problemCalls.forEach {
+                registerProblem(
+                    it,
+                    SnakemakeBundle.message("INSP.NAME.multiline.func.call"),
```

Comment:
For:
```
rule rule_148_b:
    input: foo1("text1",
                "text2", "text3")
```
2 valid froms available:
```
rule rule_148_b:
    input: foo1("text1","text2","text3")
```
and
```
rule rule_148_b:
    input: 
        foo1("text1",
                "text2", "text3")
```
At the moment the error message forces single line and it could be confusing, if 2nd variant was assumed